src/hooks/useAccountENSDomains.ts

```diff
@@ -49,7 +49,9 @@ export default function useAccountENSDomains() {
   const { accountAddress } = useAccountSettings();
   return useQuery<
     EnsAccountRegistratonsData['account']['registrations'][number]['domain'][]
-  >(queryKey({ accountAddress }), async () =>
-    fetchAccountENSDomains({ accountAddress })
+  >(
+    queryKey({ accountAddress }),
+    async () => fetchAccountENSDomains({ accountAddress }),
+    { staleTime: 60000 }
```

Comment:
What happens if a new ENS domain is registered for the account in this time?

Reply:
I don't think we need this. It should show the cached value without this and revalidate in the background. `staleTime` means that it won't be revalidated in this time.